evennia/utils/utils.py

```diff
@@ -2299,7 +2299,7 @@ def _stack_display(frame):
     return "\n".join(out[::-1])
 
 
-def m_len(target):
+def m_len(target, use_display_len=False):
```

Comment:
If you update the signature with a new kwarg, you should also update the docstring appropropriately: see the `Args:` block below.

Reply:
Updated the docstring :)